src/index.tsx

```diff
@@ -1,16 +1,16 @@
+import dotenv from 'dotenv';
```

Comment:
This is unnecessary on a react app created with CRA, since the env variables are already loaded internally by the boilerplate.